### ISSUE TITLE
Character encoding issue with GPG key

### ISSUE THREAD

**Issue by jdhoek** [IJHack/qtpass#101]
In the *Users* dialog I am encoutering some character encoding issues:

![encoding](https://cloud.githubusercontent.com/assets/683699/10044600/951795fa-61fc-11e5-9c93-38125c4292ec.png)

The public key in the screenshot belongs to someone whose name ends with `qué`, but in the user interface this is rendered as `quÃ©`. This usually means that UTF-8 text is decoded incorrectly as ISO-8859-1 text.

This public key causes no problems in Gnome's keyring or in the command line gpg tool.

My locale is all in UTF-8:

```
LANG=en_GB.UTF-8
LANGUAGE=en_GB:en
LC_CTYPE="en_GB.UTF-8"
LC_NUMERIC=nl_NL.UTF-8
LC_TIME=nl_NL.UTF-8
LC_COLLATE="en_GB.UTF-8"
LC_MONETARY=nl_NL.UTF-8
LC_MESSAGES="en_GB.UTF-8"
LC_PAPER=nl_NL.UTF-8
LC_NAME=nl_NL.UTF-8
LC_ADDRESS=nl_NL.UTF-8
LC_TELEPHONE=nl_NL.UTF-8
LC_MEASUREMENT=nl_NL.UTF-8
LC_IDENTIFICATION=nl_NL.UTF-8
```

**Comment by jdhoek** [comment]
QtPass version 1.0.1.

**Comment by annejan** [comment]
Related: https://github.com/IJHack/qtpass/issues/91 encoding issues opening or writing a password.

**Comment by annejan** [comment]
Fixed in #198
